services/galley/src/Galley/API/Teams.hs

```diff
@@ -370,6 +377,61 @@ getTeamMembers zusr tid maxResults = do
       let withPerms = (m `canSeePermsOf`)
       pure (mems, withPerms)
 
+getTeamMembersCSVH :: UserId ::: TeamId ::: JSON -> Galley Response
+getTeamMembersCSVH (zusr ::: tid ::: _) = do
+  mbZusrMembership <- Data.teamMember tid zusr
+  case mbZusrMembership of
+    Just zusrMembership ->
+      unless (isTeamOwner zusrMembership) $
```

Comment:
Don't like this one here, I think I'd like to add a hidden permission

Reply:
agreed!

Reply:
(if only to let admins in)